test/e2e/operator_test.go

```diff
@@ -63,9 +63,11 @@ func TestOperatorAvailable(t *testing.T) {
 		t.Fatal(err)
 	}
 
+	name := types.NamespacedName{Name: operatorcontroller.DNSOperatorName}
 	err = wait.PollImmediate(1*time.Second, 5*time.Minute, func() (bool, error) {
 		co := &configv1.ClusterOperator{}
-		if err := cl.Get(context.TODO(), types.NamespacedName{Name: operatorcontroller.DNSOperatorName}, co); err != nil {
+		if err := cl.Get(context.TODO(), name, co); err != nil {
```

Comment:
The log msg will look better with:
`t.Logf("failed to get deployment %s: %v", namespacedName.Name, err)`